agent/lib_aws_sdk_php.c

```diff
@@ -14,17 +14,16 @@
 #include "fw_hooks.h"
 #include "fw_support.h"
 #include "util_logging.h"
-#include "nr_segment_message.h"
-#include "nr_segment_external.h"
```

Comment:
Why the removal of these 2 includes?

Reply:
moved here: https://github.com/newrelic/newrelic-php-agent/pull/1037/files#diff-25fbc57db8661cdcd450cbe94214a9df48b6bad7c768af9ae1cdf2e01c7af0beR10-R14